src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1066,7 +1067,7 @@ private function addValidationSection(ArrayNodeDefinition $rootNode, callable $e
                             ->validate()->castToArray()->end()
                         ->end()
                         ->scalarNode('translation_domain')->defaultValue('validators')->end()
-                        ->enumNode('email_validation_mode')->values(['html5', 'loose', 'strict'])->end()
+                        ->enumNode('email_validation_mode')->values(Email::VALIDATION_MODES + ['loose'])->end()
```

Comment:
I think we must check if class `Email` exists before using it, it broken our dev-test suite on Symfony UX https://github.com/symfony/ux/actions/runs/14882498252/job/41793658394?pr=2714

Reply:
Opened https://github.com/symfony/symfony/pull/60373